lls/main.c

```diff
@@ -735,15 +761,14 @@ run_lls(struct net_config *net_conf, struct lls_config *lls_conf)
 arp:
 	if (arp_enabled(lls_conf))
 		lls_cache_destroy(&lls_conf->arp_cache);
-requests:
-	destroy_mailbox(&lls_conf->requests);
 timer:
 	rte_timer_stop(&lls_conf->scan_timer);
 stage3:
 	pop_n_at_stage3(1);
 stage2:
 	pop_n_at_stage2(1);
 stage1:
+	destroy_mailbox(&lls_conf->requests);
```

Comment:
We don't need the line `destroy_mailbox(&lls_conf->requests);` here because the mailbox is no longer initialized in `run_lls()`.